core-plugins/src/main/java/co/cask/hydrator/plugin/common/StructuredToOrcTransformer.java

```diff
@@ -120,7 +120,7 @@ private WritableComparable convertToWritable(Schema.Field field, StructuredRecor
           return new BytesWritable(Bytes.getBytes((ByteBuffer) fieldVal));
         }
       default:
-        throw new UnsupportedTypeException(String.format("{} type is currently not supported in ORC",
+        throw new UnsupportedTypeException(String.format("%s type is currently not supported in ORC",
```

Comment:
`throw new UnsupportedTypeException(String.format("Type '%s' of field '%s' is currently not supported in ORC",
                                                         fieldType.name(), field.getName()));` would be better